src/pybind/cephfs/cephfs.pyx

```diff
@@ -137,6 +137,11 @@ cdef extern from "cephfs/ceph_ll_client.h":
         timespec    stx_btime
         uint64_t    stx_version
 
+    cdef struct snap_info:
+        uint64_t id
+        char *metadata
+        unsigned long metadata_len
+
```

Comment:
nit: wouldn't `metadata_len` of type `unsigned int` suffice ?

Reply:
hmmm .. maybe just use uint64_t just like snap_id ?

Reply:
I meant to ask if a 32bit limit (4GiB) would suffice or not.
Why do we need a 64bit long data?

Reply:
normally it would suffice. I'm ok to do that if others feels so.